pkg/exporter/syslogexporter/config.go

```diff
@@ -24,10 +24,10 @@ import (
 )
 
 var (
-	unsupportedPort     = errors.New("Unsupported port: port is required, must be in the range 1-65535")
-	invalidEndpoint     = errors.New("Invalid endpoint: endpoint is required, must be a valid FQDN or IP address")
-	unsupportedProtocol = errors.New("Unsupported protocol: protocol is required, only tcp/udp supported")
-	unsupportedFormat   = errors.New("Unsupported format: Only rfc5424 and rfc3164 supported")
+	unsupportedPort     = "Unsupported port: port is required, must be in the range 1-65535"
+	invalidEndpoint     = "Invalid endpoint: endpoint is required, must be a valid FQDN or IP address"
+	unsupportedProtocol = "Unsupported protocol: protocol is required, only tcp/udp supported"
+	unsupportedFormat   = "Unsupported format: Only rfc5424 and rfc3164 supported"
```

Comment:
By convention errors are lower case :shrug:

```suggestion
	unsupportedPort     = "unsupported port: port is required, must be in the range 1-65535"
	invalidEndpoint     = "invalid endpoint: endpoint is required, must be a valid FQDN or IP address"
	unsupportedProtocol = "unsupported protocol: protocol is required, only tcp/udp supported"
	unsupportedFormat   = "unsupported format: Only rfc5424 and rfc3164 supported"
```